### ISSUE TITLE
Loss monitoring and plotting

### ISSUE THREAD

**Issue by theo1701** [zctao/omnifoldTop#61]
A road map to the development of loss monitor, with its repository found here:
https://github.com/theo1701/omnifoldTop/tree/observeLoss

After examining how loss is handled currently, I think it is totally possible to create a new loss plotter class. It will be have a global instance recording the rerun number, iteration number, and basically produce a loss histogram for all observables in each iteration (or maybe even finer resolution, for each training step).

After that, it will use the obtained data to make histograms. I'm thinking of making one for each variable at each iteration, for each rerun.

**Comment by theo1701** [comment]
Finally some checkpoints to keep trace of where thing are at (also just my personal reminder):

- [ ] check tf graph mode restriction, make sure recorder is run every time loss is triggered
- [ ] loss monitor class, init with nothing, and function to update run number
- [ ] y_true parsing, separating loss into different bins
- [ ] find a statistically sound method to "add up" the loss, I think just plain adding should be fine?
- [ ] plotting them
